en/04_Changelogs/beta/5.0.0-beta1.md

```diff
@@ -1,7 +1,68 @@
+This is the first beta release for Silverstripe CMS 5. This is a pre-release version of Silverstripe CMS 5. It is not suitable for a production environment.
+
+We are anticipating that we will tag a Silverstripe CMS 5 Release Candidate (RC) in March 2023 and a stable release in April 2023.
+
+## Overview
+- [About the Beta release](#about-the-beta-release)
+  - [Trying the beta](#trying-the-beta-trying)
+  - [Reporting bugs](#reporting-bugs)
+  - [Change freeze](#change-freeze)
+  - [Known issues](#known-issues)
+- [Upgrading your third party module](#upgrading-your-third-party-module)
+  - [Planning your upgrade](#planning-your-upgrade)
+  - [Stop using deprecated code](#stop-using-deprecated-code)
+  - [Upgrade your constraints](#upgrade-your-constraints)
+  - [Ship a Silverstripe CMS 5 version of your module](#ship-a-silverstripe-cms-5-version-of-your-module)
+- [Change to commercially supported modules](#supported-modules)
+- [Fixed dependencies](#fixed-depdencies)
+- [API changes](#api-changes)
+  - [Removed API (by module, alphabetically)](#api-removed)
+  - [General changes](#api-general)
+  - [Email](#api-email)
+  - [ORM](#api-orm)
+- [Dependency changes](#dependency-changes)
+  - [General changes](#dep-general)
+  - [TinyMCE upgraded from 4 to 6](#tinymce6)
+- [Front-end build stack upgrades](#front-end)
+  - [Webpack config](#webpack-config)
+  - [Updated javascript dependencies](#updated-javascript-dependencies)
+  - [Common upgrade patterns](#common-upgrade-patterns)
+  - [Breaking changes in _our_ javascript components or API](#js-api)
+- [Other Breaking Changes](#other-breaking-changes)
+  - [URLs generated by Silverstripe CMS don't include a trailing slash by default](#trailing-slash)
+  - [CWP Agency Extensions](#cwp-agency-extensions)
+- [Features and enhancements](#features-and-enhancements)
+  - [Other features](#other-features)
+- [Bugfixes](#bugfixes)
+
+
+## About the Beta release
+
+### Trying the beta
+
+Creating a Silverstripe CMS 5 test project is just as easy as with Silverstripe CMS 4. Simply run this command to try out the Silverstripe CMS 5 beta.
+
+```bash
+composer create-project silverstripe/installer:5.0.0-beta1 my-site
+```
+
+Silverstripe CMS 5 requires at least PHP 8.1.
+
+Review the [Server Requirements](/getting_started/server_requirements) for all the details.
+
+## Reporting bugs
+
+If you find a bugs in the beta, report it by creating an issue against the relevant GitHub repository. Prefix the title of your issue with "CMS5-beta" to make it easier for our development team to identify it.
+
+Look at [Reporting Bugs](/contributing/issues_and_bugs) for more informations.
+
+After creating an issue, maybe have a go at creating a pull request to fix it. Remember to link back your pull request to a matching issue to help the CMS development team review your work.
```

Comment:
I rewrote your suggestion to avoid "link splatter".

TLDR You should avoid pepering links through your content.

https://blog.canada.ca/2021/07/20/effective-links.html#avoid-link-splatter

Reply:
It's one link.... but okay.